wildcard/lockdown/browser/__init__.py

```diff
@@ -0,0 +1 @@
+"""Browser."""
```

Comment:
You're right. I put it there to make the pydocstyle tool happy. Later I was annoyed and disabled some checks, because it does not make sense to put "dummy" docstrings that does not add information.